index.ts

```diff
@@ -87,6 +88,15 @@ export default function viteNodeCGPlugin(pluginConfig: PluginConfig): Plugin {
         const tags = []
 
         if (config.mode === 'development') {
+            if (reactPreamble) {
+                tags.push(
+                    `<script type="module">${reactPreamble.replace(/__BASE__/g, `${dSrvProtocol}://${path.posix.join(
+                            dSrvHost,
+                            'bundles',
+                            bundleName
+                        )}/`)}</script>`
+                )
```

Comment:
According to my knowledge and information available online, in order to enable React Refresh, you have to add these lines in HTML:

```html
<script type="module">
  import RefreshRuntime from 'http://localhost:5173/@react-refresh'
  RefreshRuntime.injectIntoGlobalHook(window)
  window.$RefreshReg$ = () => {}
  window.$RefreshSig$ = () => (type) => type
  window.__vite_plugin_react_preamble_installed__ = true
</script>
```

reference: https://vite.dev/guide/backend-integration

The script lines here is very different. Is there any official documentation about this?